config/samples/tls_cluster_cr.yaml

```diff
@@ -77,11 +77,9 @@ spec:
 
     namespaces:
       - name: bar
-        memory-size: 3000000000
         replication-factor: 1
         storage-engine:
           type: device
```

Comment:
Here `type`: should be `memory`.

Reply:
I have intentionally removed the data-in-memory wherever the functionality is not related to data-in-memory